tasks/run-e2e

```diff
@@ -136,6 +140,11 @@ const args = yargs
   .option('copy-framework', { default: true, type: 'boolean', alias: 'copy' })
   .option('js-project', { default: true, type: 'boolean', alias: 'js' })
   .option('auto-start', { default: true, type: 'boolean', alias: 'start' })
+  .option('prefer-offline', {
```

Comment:
@peterp made this optional

Reply:
I think if it's causing problem let's just remove it. It's possible to set this via `.yarnrc` as well.

Reply:
I think the problem was actually the date, but I think while having prefer-offline is useful, it's better to not use it by default (to keep the installation more accurate to the user's flow).

Reply:
Yeah, let's take it out completely. It's mostly useless if it's not on by default, and can be overwritten by a user's own configuration.

The problem was the date?

Reply:
Having `:` in the folder name :) https://github.com/redwoodjs/redwood/pull/1932/files#diff-e4d8d48827bb7da131ab93737cf7a718801e74e8c86b22e011f27dd4f948b1edR174